src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -19,42 +18,50 @@ The following options can be configured on the root level of your browser-based
+| networkCaptureBodies   | boolean              | `true`  | Decide whether to capture request and response bodies for URLs defined in `networkDetailAllowUrls`                                                                                                |
+| networkRequestHeaders  | string[]             | `[]`    | Capture request headers in addition to default headers.                                                                                                                         |
+| networkResponseHeaders | string[]             | `[]`    | Capture response headers in addition to default headers.                                                                                                                        |
 
-## Identifying Users
+### Sampling
+
+If you want to understand how exactly our session sampling works, check out our <PlatformLink to="/session-replay/understanding-sessions/#default-session-initialization">Understanding Sessions > Default Session Initialization</PlatformLink> documentation.
+
+### Network Details
 
-You can use the Sentry SDK to set the user of a session. To associate a user identity to a session replay, refer to the <PlatformLink to="/enriching-events/identify-user">Identifying User Docs</PlatformLink>.
+By default, Replay will capture basic information about all outgoing fetch and XHR requests in your application. This includes the URL, request and response body size, method, and status code. The intention is to limit the chance of collecting private data by default.
+
+To capture additional information such as headers or bodies, you'll need to opt-in via `networkDetailAllowUrls`. This gives you control to add only URLs that are safe to capture bodies, avoiding any endpoints that can contain personal identifiable information.
+
+Any URL matching the given pattern(s) will then be captured with additional details:
 
 ```javascript
-Sentry.setUser({ email: "jane.doe@example.com" });
+new Replay({
+   networkDetailAllowUrls: ['https://sentry.io/api']
+});
 ```
 
-## Start and Stop Recording
+If you give us a string, we'll match any URL that contains that string. You can use a Regex to handle exact or more complex matches.
 
-Replay recording starts if it's included in the `integrations` array when calling `Sentry.init` or, when `addIntegration` is called from a Sentry client instance. To stop recording, call `stop()`:
+Requests to a URL matched by the configured patterns will be enhanced with the request and response body, as well as the default headers `content-type`, `content-length`, and `accept, by default. If you want to capture additional headers, you'll have to configure them with the options `networkRequestHeaders` and `networkResponseHeaders`.
 
-```javascript
-const replay = new Sentry.Replay();
+Note that captured bodies will be truncated to a maximum size of 150k characters. Content in bodies will be PII sanitized server-side, based on object keys and values.
```

Comment:
```suggestion
Note: captured bodies will be truncated to a maximum size of 150k characters. Content in bodies will be PII sanitized server-side, based on object keys and values.
```

Reply:
Can we stylized this "Note" like how we do "Unlike [sessions](https://docs.sentry.io/product/releases/health/#session) on the Releases page, user sessions in Session Replay can span multiple page loads." on [this page](https://docs.sentry.io/product/session-replay/)?

Reply:
```suggestion
<Note>
Captured bodies will be truncated to a maximum size of 150k characters. Content in bodies will be PII sanitized server-side, based on object keys and values.
</Note>
```